microservice-transaction-sample/rpc/build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
     id 'java'
     id 'java-library-distribution'
-    id 'com.google.protobuf' version '0.9.1'
+    id 'com.google.protobuf' version '0.9.4'
```

Comment:
To suppress the following warning message, `com.google.protobuf` must be upgraded.
```
The Provider.forUseAtConfigurationTime method has been deprecated. This is scheduled to be removed in Gradle 9.0. Simply remove the call. Consult the upgrading guide for further information: https://docs.gradle.org/8.5/userguide/upgrading_version_7.html#for_use_at_configuration_time_deprecation
```